spec/rails_app/config/initializers/audited.rb

```diff
@@ -0,0 +1,5 @@
+require 'audited'
+
+Audited.config do |config|
+  config.max_audits = 10
```

Comment:
This might lead to unexpected results in future tests, perhaps only mock this for tests related to this option?